cli/crates/common/src/environment.rs

```diff
@@ -325,18 +324,38 @@ fn find_grafbase_configuration(path: &Path, warnings: &mut Vec<Warning>) -> Opti
 pub fn add_dev_dependency_to_package_json(project_dir: &Path, package: &str, version: &str) -> Result<(), CommonError> {
     let package_json_path = project_dir.join(PACKAGE_JSON_NAME);
 
-    if !package_json_path.exists() {
-        run_npm_init(project_dir)?;
-    }
-
-    let file = fs::File::open(&package_json_path).map_err(CommonError::AccessPackageJson)?;
-
-    let Ok(Value::Object(mut package_json)) = serde_json::from_reader(&file) else {
+    let mut package_json = if !package_json_path.exists() {
+        let name = project_dir
+            .file_name()
+            .map(|it| it.to_string_lossy())
+            .unwrap_or(Cow::Borrowed("grafbase-project"));
```

Comment:
The name is not sanitized in any way. We could slugify it but it does not matter for our workflow. `npm install` succeeds even with the invalid name.